library/network/src/modules/SuSEFirewall.rb

```diff
@@ -38,6 +38,9 @@ class SuSEFirewallClass < Module
 
     include Yast::Logger
 
+    YES = "yes"
+    NO  = "no"
```

Comment:
it is not strange, it say what it should. You set true to given settings. Of course in perfect OOP world, it will be object and set it with enable/disable